src/components/content/link.tsx

```diff
@@ -50,10 +50,14 @@ export function Link({
 
   const isAbsolute = href.indexOf('//') > -1
   const isExternal = isAbsolute && !href.includes('.serlo.org')
-  const isLegacyLink = legacyLinks.indexOf(displayHref) > -1
+  const isLegacyLink =
+    legacyLinks.indexOf(displayHref) > -1 ||
+    displayHref.startsWith('/user/profile/') ||
+    displayHref.startsWith('user/profile/')
```

Comment:
Why could both of these cases occur?

Reply:
because authors… and currently we don't clean up the link for this check. 
since this is a temporary solution I'd keep it this way.

Reply:
Welp, that wouldn't work correctly anyways with our alias system ^^'. But okay, if it's the actual href anyways, I'm fine with having this as a quick fix until we sync this.